docs/core/diagnostics/microsoft-diagnostics-netcore-client.md

```diff
@@ -73,21 +73,29 @@ public EventPipeSession StartEventPipeSession(
     IEnumerable<EventPipeProvider> providers,
     bool requestRundown = true,
     int circularBufferMB = 256);
+public Task<EventPipeSession> StartEventPipeSessionAsync(
```

Comment:
I've added it here, directly under the sync overload, to avoid copy/paste all the docs, but let me know if you want that.